data/vim/doc/app/vifm-app.txt

```diff
@@ -6744,5 +6746,96 @@ Website: https://vifm.info/
 Q&A: https://q2a.vifm.info/
 Wiki: https://wiki.vifm.info/
 
+--------------------------------------------------------------------------------
+*vifm-cmd-quoting*
+
+Parsing of text after the colon is based on quoting and escaping rules. These
+rules determine how Vifm splits the entered text into arguments. (Note: some
+commands, such as :echo, interpret their arguments as an expression, but the
+quoting rules themselves remain the same.)
+
+Supported mechanisms:
+
+  - single quotes '...'
+    The content is taken literally.
+    Escaping does not work inside.
+    Everything between the quotes becomes a single argument.
+
+    Example:
+        :echo 'a\ b'
+        output: a\ b
+
+  - double quotes "..."
+    Also form a single argument.
+    A backslash inside double quotes escapes the next character
+    (for example \" becomes a quote, \\ becomes a backslash).
+    Escape sequences are not interpreted.
+
+    Example:
+        :echo "a b\ c \"x\""
+        output: a b c "x"
+
+  - backslash escaping \
+    Outside quotes, it allows including spaces, quotes, or a backslash
+    into an argument.
+    Inside double quotes it works the same way.
+    It does not work inside single quotes.
+
+    Example (visible behavior):
+        :command Foo echo a\ b
+        defines the Foo command with a single argument: a b
+
+        Calling it:
+            :Foo
+        output:
+            a b
+
+After processing quotes and backslashes, the line is split into arguments by
+spaces. Quotes and escaping backslashes do not appear in the resulting
+arguments.
+
+--------------------------------------------------------------------------------
+*vifm-name-conflicts*
+
+When copying or moving files, Vifm can encounter a name conflict: a file
```

Comment:
This is relevane to creating symbolic links as well (<kbd>al</kbd>, <kbd>ar</kbd>).